cmd/minikube/cmd/delete.go

```diff
@@ -137,7 +142,8 @@ func runDelete(cmd *cobra.Command, args []string) {
 	}
 
 	if deleteAll {
-		deleteContainersAndVolumes()
+		deleteContainersAndVolumes(oci.Docker)
```

Comment:
maybe do them in parallel ?

Reply:
possibly, the theory is that one of them will be "empty"